src/components/plots/gridPlot.ts

```diff
@@ -43,28 +48,49 @@ export module Plot {
     }
 
     /**
-     * @param {string} attrToSet One of ["x", "y", "fill"]. If "fill" is used,
+     * @param {string} attrToSet One of ["x", "y", "x2", "y2", "fill"]. If "fill" is used,
      * the data should return a valid CSS color.
      */
     public project(attrToSet: string, accessor: any, scale?: Scale.AbstractScale<any, any>) {
+      attrToSet = attrToSet === "x1" ? "x" : attrToSet;
```

Comment:
? I thought with our discussion, x would map to x1/x2 if the scale is category.  Now we have x1 map to x?  This also is a bit confusing since RectanglePlot wants to be in terms of x1/x2 to projecting on the x attribute is not useful for RectanglePlot.